content/concepts/websockets.textile

```diff
@@ -427,7 +427,7 @@ h3(#socketio). Socket.io
 
 "Socket.io":https://github.com/socketio/socket.io has been around for a while now and could be thought of as the "jQuery" of WebSockets. It uses "long polling":/concepts/long-polling and WebSockets for its transports, by default starting with "long polling":/concepts/long-polling and then upgrading to WebSockets if available. Given the waning relevance of "long polling":/concepts/long-polling, Socket.io's main drawcards these days are its "other features":https://stackoverflow.com/questions/38546496/moving-from-socket-io-to-raw-websockets/38546537#38546537, such as restoring dropped connections, automatic support for JSON, and "namespaces", which are essentially isolated messaging channels multiplexed over the same client connection.
```

Comment:
Can you please replace this "Socket.io":https://github.com/socketio/socket.io with "Socket.io":https://ably.com/concepts/socketio